app/models/solana/stake_account.rb

```diff
@@ -60,6 +60,7 @@ def initialize(args)
     # Gets the output from the Solana CLI and populates the attributes
     def get
       ssa = cli_request("stake-account #{@address}", @rpc_urls)
+      @cli_error = ssa[:cli_error]
```

Comment:
Interesting...How come you're accessing this key as a symbol but all the others as a string? Some consistency would be nice.